swagger.yml

```diff
@@ -2025,8 +2049,8 @@ paths:
           $ref: '#/components/responses/error'
   /alerts:
     get:
-      summary: search alerts using the filters in the request
-      description: search alerts using the filters in the request
+      summary: search alerts
+      description: 'This API fetches all alerts. The parameters in version 2.0 are not used since all filters are executed by the frontend. This API is also called by the Dashboard. '
```

Comment:
'This API is also called by the Dashboard' is no longer true.

Reply:
![Screenshot_20200713_202624](https://user-images.githubusercontent.com/53040049/87340057-dc718b00-c547-11ea-9464-24dd84eb39f1.png)

Reply:
What I was trying to say is that the API alerts it's called to build the Dashboard Page. Never mind I removed the comment.